docs/home/howto/UPGRADE.md

```diff
@@ -109,11 +109,7 @@ To upgrade the cluster components run the following steps:
     epicli prepare --os OS
     ```
 
-   Where OS should be `centos-7`, `redhat-7`, `ubuntu-18.04`. This will create a directory called `prepare_scripts` with
-   the following files inside:
-
-    - download-requirements.sh
-    - requirements.txt
+   Where OS should be `centos-7`, `redhat-7`, `ubuntu-18.04`. . This will create a directory called `prepare_scripts` with the needed files inside.
```

Comment:
```suggestion
   Where OS should be `centos-7`, `redhat-7`, `ubuntu-18.04`. This will create a directory called `prepare_scripts` with the needed files inside.
```